### DISCUSSION TITLE
How to extend the image with mirror image at the border.

### DISCUSSION THREAD

**Discussion by uraxurax** [ImageMagick/ImageMagick#5881] [Help] [answered]
I want to extend the image with mirror image at the border.
Is it possible by using ImageMagick?

Following the image sample

Original Image
![test](https://user-images.githubusercontent.com/9453589/207898091-4269ee57-afe0-4f56-af63-57a22792f973.jpg)

Extension Image with 32 pixel perimeter 
![test_pad](https://user-images.githubusercontent.com/9453589/207898118-caa12e4a-5a87-43fd-90ae-ae779d3f4cb7.jpg)

**Comment by snibgo** [comment, marked as answer]
How about this, where I have named your input `flowerPots.jpg`:
```
magick flowerPots.jpg -filter point -virtual-pixel Mirror -define distort:viewport=2100x1900-100-100 -distort SRT 1,0 out.png
```
Adjust the viewport width, height, x-offset and y-offset as you want.

**Reply by uraxurax** [reply]
Thank you!
It works perfect!
